README.rst

```diff
@@ -736,22 +735,21 @@ Technical outline of how cwltool works internally, for maintainers.
       there is currently no work ready to run) or end of iteration (indicating
       the process is complete.)
    #. Invoke the runnable item by calling ``run()``.  This runs the tool and gets output.
-   #. Output of a process is reported by an output callback.
+   #. An output callback reports the output of a process.
    #. ``job()`` may be iterated over multiple times.  It will yield all the work
       that is currently ready to run and then yield None.
 
 #. ``Workflow`` objects create a corresponding ``WorkflowJob`` and ``WorkflowJobStep`` objects to hold the workflow state for the duration of the job invocation.
 
    #. The WorkflowJob iterates over each WorkflowJobStep and determines if the
       inputs the step are ready.
-   #. When a step is ready, it constructs an input object for that step and
+   #. When a step is ready, it constructs an input object and
```

Comment:
Please leave that as it was

Reply:
Per your request, I reverted to the original wording.